whylog/tests/tests_client/test_backtrackerSearcher.py

```diff
@@ -0,0 +1,62 @@
+from unittest import TestCase
+import os.path
+from whylog.tests.tests_client.constants import test_paths, AFewLinesLogParams
+
+from whylog.client import searchers
+
+
+class TestBacktrackSearcher(TestCase):
+    def _get_log_file_path(self):
+        prefix_path = os.path.join(*test_paths.path_test_files)
+        path = os.path.join(prefix_path, "a_few_lines.log")
```

Comment:
if method name clearly specifies what return value is, don't assign result to a variable just to return it in the next line. `return os.path.join(prefix_path, "a_few_lines.log")` would be more concise.